File: PHP Excel Examples/WorksheetsApiDemo/NamedRanges/getNamedRanges.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "GetNamedRanges.xlsx";
$folder= "input";
$storage = null;
$result = $apiInstance->getNamedRanges($name, $folder, $storage);

File: Python Excel Examples/WorksheetsApiDemo/named_ranges/getNamedRanges.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "GetNamedRanges.xlsx"
storage = ""
folder = "ExcelDocument"
result=api.get_named_ranges(name, folder=folder, storage=storage)